docs/source/projects.rst

```diff
@@ -374,7 +374,7 @@ in the Databricks docs (`Azure Databricks <https://docs.microsoft.com/en-us/azur
 
 .. _kubernetes_execution:
 
-Run an MLflow Project on Kubernetes (experimental)
```

Comment:
Let's keep this as `experimental` until we've fixed some of the many reported issues